lib/tls.js

```diff
@@ -189,8 +183,7 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
 
   if (!valid) {
     var err = new Error(
-        util.format('Hostname/IP doesn\'t match certificate\'s altnames: %j',
-                    reason));
+        `Hostname/IP doesn't match certificate's altnames: "${reason}"`);
```

Comment:
It's because of the %j (JSON.stringify) it replaces.  `reason` is always a string so putting it in quotes looks right to me.